sycl/source/detail/memory_manager.cpp

```diff
@@ -176,27 +175,40 @@ void copyH2D(SYCLMemObjT *SYCLMemObj, char *SrcMem, QueueImplPtr SrcQueue,
   DstAccessRange[0] *= DstElemSize;
   SrcSize[0] *= SrcElemSize;
   DstSize[0] *= DstElemSize;
-
-  cl_command_queue CLQueue = UseExclusiveQueue
-                                 ? TgtQueue->getExclusiveQueueHandleRef()
-                                 : TgtQueue->getHandleRef();
-
+  
   if (1 == DimDst && 1 == DimSrc) {
     CHECK_OCL_CODE(clEnqueueWriteBuffer(
-        CLQueue, DstMem,
+        TgtQueue->getHandleRef(), DstMem,
```

Comment:
I'd prefer we maintain the usage of the CLQueue variable unless there is good reason